src/hooks/useTriggerOrdersFormInputs.ts

```diff
@@ -60,12 +60,12 @@ export const useTriggerOrdersFormInputs = ({
         },
         {
           field: TriggerOrdersInputField.stopLossPrice,
-          value: stopLossOrder.triggerPrice,
+          value: stopLossOrder.triggerPrice?.toString(),
           hasFormInput: true,
         },
         isLimitOrderType(stopLossOrder.type) && {
           field: TriggerOrdersInputField.stopLossLimitPrice,
-          value: stopLossOrder.price,
+          value: stopLossOrder.price.toString(),
```

Comment:
There is a risk of losing precision when using native number toString() but it's what was already happening so I don't think it can cause bugs.

Reply:
Yeah that might be the correct play here. I will make a helper function that does safe number to string.

Reply:
Looks like number to string and bignumber to string both do scientific notation above/below certain thresholds so I don't think we gain much unless we do a bunch of bignumber overrides.